en-US/Conductors_Companion/09_Magic_Items/Magic_Item_Descriptions/A/Ammunition_plus.md

```diff
@@ -0,0 +1,35 @@
+#### Ammunition, +1, +2, or +3
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:1:item_name]:- "Ammunition, +1"
```

Comment:
This is basically 3 types of item modification rolled into one. So I came up with a way to have details about multiple items in a single file: We're adding a counter to the metadata.
I _think_ it should be relatively simple to build a script that can handle this syntax in addition to the "basic" syntax needed in most items.